### ISSUE TITLE
Lantern Engine refuses to load correctly

### ISSUE THREAD

**Issue by MarvinG6547** [michael-fadely/sadx-dc-lighting#89]
#### Encountered behavior

Every single time without fail, no matter of what mods I have active or what I do, Lantern Engine spits out the error "DLL error -%1 is not a legal Win32-Application"
I installed DirectX Web Runtime, I installed .Net 7.0 for the mod launcher and I even reinstalled DirectX 9 itself, but nothing fixes the issue. I've been at this for an hour now and I'm going to go insane.
(Yes I did update it, yes I updated the mod loader. Everything should be in perfect working order)

#### Installed mods

* ADX Audio
* Dreamcast Conversion
* Frame Limit
* HD GUI 2
* Idle Chatter
* Input Mod
* Onion Skin Blur
* Smooth Camera
* Sound Overhaul
* Time of Day

#### Steps to reproduce:
1. Literally just try launching the game with Lantern Engine enabled

**Comment by PiKeyAr** [comment]
Do you have the latest Visual C++ redist?

**Comment by MarvinG6547** [comment]
Yes, I have every Visual C++ Rediststributable ranging from 2005 to 2022
![image](https://github.com/michael-fadely/sadx-dc-lighting/assets/117035792/3136b414-7d34-4da2-a0bb-e82881012b6c)

**Comment by ghost** [comment]
I did find _a_ solution to this, but it's a bit of an awkward, hacky-feeling fix:

1. Download [this](https://www.microsoft.com/en-us/download/details.aspx?id=8109) installer.
2. `directx_Jun2010_redist.exe` is seemingly just a self-extracing archive, so you can either run it or open it in 7Zip (or something similar) - I took the latter option.
3. Extract `Jun2010_d3dx9_43_x86.cab`, and open it in 7Zip; you should see `d3dx9_43.dll`.
4. Drop this file into the folder that SADX is installed to; specifically, the folder also containing `sonic.exe`.
5. Set the Renderer Backend to Direct3D 9 in SA Mod Manager's Graphics settings tab.

This worked for me, but might not be the problem you're having - I'm sorry, but I probably wouldn't be able to help in that case.

**Comment by MarvinG6547** [comment]
I already solved this ages ago with a visual c++ aio installer, [this one](https://github.com/abbodi1406/vcredist/releases) if I remember correctly, but thanks for sharing your solution!
